.changeset/olive-bags-drive.md

```diff
@@ -0,0 +1,18 @@
+---
+"@astrojs/solid-js": minor
+---
+
+Adds a `devtools` option
+
+You can enable the [official Solid Devtools](https://github.com/thetarnav/solid-devtools) while working in development mode by setting `devtools:true` in your `solid()` integration config:
```

Comment:
I think we should also document that you need to install `solid-devtools` too (and in the docs)

Reply:
You're right, I'm adding it!